pwa/contexts/RepairerRegistrationContext.tsx

```diff
@@ -71,6 +73,8 @@ const initialValue = {
   isRoving: false,
   success: false,
   successMessage: '',
+  counter: 1,
```

Comment:
Use a meaningful variable name like `signNumber` (numéro de l'enseigne) par exemple.

Reply:
Or why not using the name of the shop? To have:
```
<Typography
            variant="h2"
            textAlign="center"
            color="primary.main"
            pb={2}>
            Mon enseigne
            {counter <= 1 ? 'Mon enseigne' : `Enseigne ${counter}`} // Enseigne Foo
          </Typography>
```

Reply:
I can't use the name because at this point, the shop is not yet created.